ubuntudesign/vanilla-framework
# Update lists

# Done
- Added Sass doc comments
- Wrapped varieties in mixins
- Updated demo to reflect available lists

## QA
- Run ```gulp build```
- Run ```google-chome index.html``` and check the list section looks ok.
- Run ```google-chome build/docs/index.html``` and check the doc for lists contain all the versions.
